### PR TITLE
Application Logo View

### PR DESCRIPTION
**Pull request by zahnno**
CPF-3249 #time 1h
- Implemented a generic application logo view.
- Added a `largeIcon` to `theme`.
- Utilizes `theme` `logo` and `appName` or `largeLogoIcon` if theme has it enabled.
- Replaced BusinessLogoView on TopNavigationView with ApplicationLogoView.

Example:

Screen Shot 2019-10-29 at 2 34 54 PM

### CODE REVIEW

File: src/foam/nanos/u2/navigation/ApplicationLogoView.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+foam.CLASS({
```
Comment (kgrgreer) [inline src/foam/nanos/u2/navigation/ApplicationLogoView.js:6]: Copyright header.

File: src/foam/nanos/u2/navigation/ApplicationLogoView.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        .addClass(this.myClass())
+        .on('click', this.goToDefault)
+          .start({
+            class: 'foam.u2.tag.Image',
+            data$: this.theme$.dot('logo')
+          })
+            .hide(this.theme$.dot('largeIconEnabled'))
+            .addClass('logo')
+            .start('span').addClass('appName').add(this.theme$.dot('appName')).end()
+          .start({
```
Comment (kgrgreer) [inline src/foam/nanos/u2/navigation/ApplicationLogoView.js:50]: Are you start()'s and end()'s balanced and indented properly?

**Review by jlhughes — APPROVED** [review]

**Comment by kgrgreer** [comment]
Has merge conflicts.

**Comment by Nauna** [comment]
addresses: https://nanopay.atlassian.net/browse/CPF-3249

**Pull request merged**